_posts/2022-02-28-war-in-ukraine.md

```diff
@@ -0,0 +1,29 @@
+---
+layout: post
+title: "The War in Ukraine"
+author: [michal.kosmulski]
+tags: [tech, "soft skills", helping]
+---
+The attack on Ukraine has shocked the whole world, but in Poland it is perceived in an especially poignant way.
+Not only is Ukraine our direct neighbor (as are Russia and Belarus), but Poland’s difficult history allows us to
+sympathize with the Ukrainians even more. Many live in Poland, so it’s affecting people we know personally.
+
+While the fighting continues, here’s what [Allegro](https://allegro.tech) is doing, and what you can do, too, to help
+humanitarian efforts.
+
+* If you know any Ukrainians, your coworkers in particular, talk to them and show your support. Even minor gestures are
+  important. Ask directly what kind of help they might need rather than trying to guess.
+* Do not marginalize Russians and Belorussians. People are not the same as their governments, and often do not support
+  their states’ actions.
+* Work with your HR people and consider what kind of help you can offer as a company. Some examples would be extra days
+  off, legal and financial support for bringing families to safety, or offering psychological support.
+* On a personal level, consider supporting one of the trustworthy humanitarian organizations. You can support them
+  with cash or by donating necessary items which are being collected in most cities.
```

Comment:
```suggestion
* On a personal level, consider supporting one of the trustworthy humanitarian organizations. You can support them
  with cash, volunteer work or by donating necessary items which are being collected in most cities.
```

Dorzuciłabym jeszcze coś w stylu "You can check out your local authorities' websites to find out what kind of help is needed right now and how else you can get involved".

Reply:
Dodałbym tutaj punkt: "Be extremely cautious of fake fundraising campaigns that are popping up on every fundraising platform. Carefully check the fundraiser. Organisations known for their credibility and ability to sped the money effectively are Polska Akcja Humanitarna [https://siepomaga.pl/pah-ukraina] and Polskie Centrum Pomocy Międzynarodowej, who you can safely support through Allegro [https://allegro.pl/charytatywni/polskie-centrum-pomocy-miedzynarodowej/cele/pomoc-dla-obywateli-ukrainy].

Reply:
Dobra uwaga, choć raczej nie będę wymieniał konkretnych nazw żeby nie było, że inne z założenia nie są (choć oczywiscie do tych dwóch zastrzeżeń nie mam, PAH nawet wspieram prywatnie)

Reply:
Może warto podrzucić też link tutaj https://demagog.org.pl/analizy_i_raporty/support-ukraine-list-of-credible-fundraisers-and-campaigns/ gdzie jest szeroka lista potwierdzonych akcji